src/app/(dashboard)/temas/[id]/results/page.tsx

```
@@ -0,0 +1,219 @@
+            }`}
+          >
+            <div className="font-medium">
+              {feedback.isCorrect ? 'Resposta Correta!' : 'Resposta Incorreta'}
+            </div>
+            {feedback.explanation && (
+              <div className="mt-2">
+                <div className="text-sm font-medium">Explicação:</div>
+                <div
+                  className="mt-1 text-sm"
+                  dangerouslySetInnerHTML={{
+                    __html: renderContent(feedback.explanation),
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)